### ISSUE TITLE
NVDA remote

### ISSUE THREAD

**Issue by bedenhofer** [nvaccess/nvda#10068]
#### Other information about your system:

#9958 ## Other questions
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes I have, and no behaviors, whatsoever NVDA remote doesn't work on the beta version, whitch is python 3

**Comment by XLTechie** [comment]
Duplicate of #10067

**Comment by Adriani90** [comment]
Closing as duplicate

**Comment by bedenhofer** [comment]
Duplicate of # This needs to be addressed, that's why, I reported it

**Comment by XLTechie** [comment]
We are not able to help you with this. NVDA Remote is a separate project, not 
part of NVDA.

I told you where and to whom you should report it.
See my response to https://github.com/nvaccess/nvda/issues/10067

**Comment by Brian1Gaff** [comment]
If anyone has issues filling out the template directly on line. Perhaps what 
I do may help.
 I cut and paste the template into a text file offline.
 I can then fill it in at my leisure while trying stuff. Then when I'm 
ready, I go on, create the  ticket,  delete the template online and paste my 
completed one in instead. Saves a lot of hassle I find.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----
